### DISCUSSION TITLE
iTwin Sychronizer doesn't find reference files in sub directories

### DISCUSSION THREAD

**Discussion by JohnMajerle** [iTwin/itwinjs-core#1848] [Q&A]
The **iTwin Sychronizer** isn't finding references files located in sub directories.  For my test I am bridging a file located in the directory:

`C:\FDOTConnect\Workspaces\FDOT\WorkSets\0_WORKSET_TEMPLATE\
`
The reference attachments are:

![image](https://user-images.githubusercontent.com/30668739/125459022-22f7d0c6-f841-4108-9843-6e5b2381b2e5.png)

I defined my **iTwin Sychronizer** _File search paths_ as follows:

```
C:\FDOTConnect\Workspaces\FDOT\WorkSets\0_WORKSET_TEMPLATE
C:\FDOTConnect\Workspaces\FDOT\WorkSets\0_WORKSET_TEMPLATE\Roadway
C:\FDOTConnect\Workspaces\FDOT\WorkSets\0_WORKSET_TEMPLATE\Signing
C:\FDOTConnect\Workspaces\FDOT\WorkSets\0_WORKSET_TEMPLATE\Survey
```
The synchronization only finds the master file and the dgns located in the same directory as the master file.

What is the proper way to set this up?

Thanks,

John M.

**Comment by calebmshafer** [comment]
@jffmarker ?

**Reply by sneinamdr** [reply]
@JohnMajerle I confirmed this is a bug on our side and we will have it fixed and let you know once its in a release.

**Reply by sneinamdr** [reply]
@JohnMajerle we further explored and tried creating a dataset that has a structure like one you mentioned above but we cannot reproduce the issue. ITSYN was able to sync the master file and its attached reference files located in the subfolders. We only had to set the search path to the directory of the master file. Can you provide further details or steps on how this dataset was created? Also why do you have all the sub folder paths in the search path text box?
